src/app/about/aboutUs.module.css

```diff
@@ -0,0 +1,17 @@
+.container {
+  width: 100%;
+  max-height: calc(100vh - var(--header-height) - var(--footer-height));
+  padding: 4rem;
+  overflow: scroll;
```

Comment:
thanks for making it scrollable. Coul you please also limit the width on desktop with help of `--max-layout-width` variable (for the contant form also).